py/desisim/scripts/quickquasars.py

```diff
@@ -433,7 +489,8 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
             DZ_FOG = DZ_FOG[indices]
             nqso = args.nmax
             
-    if args.dn_dzdm is not None:
+    if args.dn_dzdm:
+        # TODO: Deprecate this option. It will be faster to do this on preprocessing.
```

Comment:
You can add `warnings.warn("dn_dzdm will be deprecated. It is faster to do this in preprocessing", DeprecationWarning)`